dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/config/SpringConfigBackendDatabase.java

```diff
@@ -66,6 +66,7 @@ public Flyway flyway(DataSource pds) {
             .locations(
                 "classpath:/de/digitalcollections/cudami/server/backend/impl/database/migration")
             .baselineOnMigrate(true)
+            .outOfOrder(true)
```

Comment:
Wofür ist das gut?

Reply:
dass man in einem branch evtl schon mit einer migration version 3.0 beginnen kann, während in einem anderen branch noch auf 2.x weiterentwickelt werden kann ohne dass man sich mit den migrationen in die quere kommt. generell zu empfehlen